src/assembler.cpp

```diff
@@ -1242,6 +1242,9 @@ int main(int argc, char* argv[])
                         constant.b = (uint32_t)values[0] != 0;
                         break;
                     }
+
+                    default:
```

Comment:
@Dettorer Sorry, I'm not quite following your logic here. We can discuss this matter further another time, but please drop this change from the PR for the time being.